### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by liuxingchui**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Richard-Cao/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# ReviewNinja-Welcome
```
Comment (Richard-Cao) [inline ReadMe.md]: test

Comment (Richard-Cao) [inline ReadMe.md]: hahaha

Comment (Richard-Cao) [inline ReadMe.md]: !fix aaa

Comment (Richard-Cao) [inline ReadMe.md]: !fixed test

Comment (bugknightyyp) [inline ReadMe.md]: 6666

**Pull request merged**